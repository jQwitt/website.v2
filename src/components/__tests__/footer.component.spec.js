import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Footer from '../footer.component';

afterEach(cleanup);
describe('Footer renders correctly', () => {
  test('snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
