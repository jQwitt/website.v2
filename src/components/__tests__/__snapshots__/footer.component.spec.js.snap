// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly snapshot 1`] = `
<DocumentFragment>
  <div
    class="col-12 text-center content-footer"
  >
    <p>
      Font generated by
      <a
        href="https://www.flaticon.com"
      >
         flaticon.com
      </a>
    </p>
    <p>
      Under
      <a
        href="http://creativecommons.org/licenses/by/3.0/"
      >
         CC,
      </a>
      <a
        data-file="017-right-arrow"
        href="https://www.flaticon.com/authors/smashicons"
      >
        Smashicons,
      </a>
      <a
        data-file="019-coding-1"
        href="https://www.flaticon.com/authors/becris"
      >
        Becris
      </a>
    </p>
  </div>
</DocumentFragment>
`;
